crates/iota/src/client_commands.rs

```diff
@@ -243,22 +234,25 @@ pub enum IotaClientCommands {
         #[clap(flatten)]
         opts: OptsWithGas,
     },
-
-    /// Generate new address and keypair with keypair scheme flag {ed25519 |
-    /// secp256k1 | secp256r1} with optional derivation path, default to
-    /// m/44'/4218'/0'/0'/0' for ed25519 or m/54'/4218'/0'/0/0 for secp256k1
-    /// or m/74'/4218'/0'/0/0 for secp256r1. Word length can be { word12 |
-    /// word15 | word18 | word21 | word24} default to word12 if not specified.
+    /// Generate new address and keypair with optional key scheme {ed25519 |
+    /// secp256k1 | secp256r1} which defaults to ed25519, optional alias which
+    /// defaults to a random one, optional word length { word12 | word15 |
+    /// word18 | word21 | word24} which defaults to word12, and optional
+    /// derivation path which defaults to m/44'/4218'/0'/0'/0' for ed25519,
+    /// m/54'/4218'/0'/0/0 for secp256k1 or m/74'/4218'/0'/0/0 for secp256r1.
     #[clap(name = "new-address")]
     NewAddress {
+        #[clap(long, default_value_t = SignatureScheme::ED25519)]
         key_scheme: SignatureScheme,
         /// The alias must start with a letter and can contain only letters,
         /// digits, hyphens (-), or underscores (_).
+        #[clap(long)]
         alias: Option<String>,
+        #[clap(long)]
         word_length: Option<String>,
+        #[clap(long)]
         derivation_path: Option<DerivationPath>,
```

Comment:
Could add actual defaults to these since they all default.

Reply:
And I think it would appear in the generated help text too, so you could remove it from the comment.

Reply:
The default depends on the key_scheme so it's not straightforward

Reply:
At least for `derivation_path`, the others are selected somewhere else not even in this crate so I'm not sure I would change it tbh

Reply:
alright!